defaults-fr.yml

```diff
@@ -236,7 +236,7 @@ categories:
             default: true
             text: Screenshot 2020-01-09 at 13.27.20.png
           - value: false
-            text:
+            text: |
```

Comment:
I'm not sure on why this is necessary?

Reply:
The current yaml syntax is a list of strings, while the schema expects a string. This turns it into a normal string (which is then processed as markdown). I'm interpreting the intent here, might have gotten something wrong.